lib/model.js

```diff
@@ -40,6 +40,10 @@ var Utils = require('./utils')
  */
 class Model {
 
+  static get Instance() {
+    return this;
+  }
```

Comment:
I added it because I didn't touch any other files and didn't know wether the tests would pass if I remove it (because it might be used in other files). I'm happy to remove it, you can always access the model class through `.constructor`